README.md

```diff
@@ -41,34 +38,100 @@ require('apostrophe')({
   }
 });
 ```
+## Exporting Files
+
+### Pages
+![Screenshot highlighting the export menu item in the context menu of a page in the manager.](images/page-export.png)
+
+Pages can currently only be exported singly. When exporting a page it will retain its publication status upon import. A page that is published when exported will be published on import, while draft pages will remain in draft status. In order to export a page, you use the context menu to the right of each page in the manager.
+
+Exporting the parent page in a set of nested documents will **not** export the child pages. Each child page must be exported separately. The parent page must already exist, or be separately exported and then subsequently imported, for any child page to appear in the page manager of the import project.
```

Comment:
Support for exporting multiple pages at once is planned for a future release.

Reply:
Added.